library.js

```diff
@@ -36,7 +36,7 @@ const constants = {
   displayName: 'Discord',
   admin: {
     route: '/plugins/sso-discord-alt',
-    icon: 'fa-pied-piper'
+    icon: 'fa-discord'
```

Comment:
Does node-bb now ship with the version of Font Awesome that includes this?

Reply:
Yep, at least my installation did... might be good to double check if you want to be certain, i installed ver 1.18.x initially but since updated to 1.19

Reply:
Can you please provide a link to the node-bb source showing the version of FA they bundle? I do not have time to research such things myself.

Reply:
Sure,
https://github.com/NodeBB/NodeBB/blob/05e2b3541ba8c06907d2e17b71b90170db8c1720/public/vendor/fontawesome/webfonts/fa-brands-400.svg?short_path=4e48a46&L=1593#L1593

You can see here that there is in fact a discord glyph

Reply:
Thank you. All that is missing from this PR is for the Readme to be updated.

Reply:
I'll get on it then